_profiles/v2-article.html

```diff
@@ -38,6 +38,10 @@
 #
 budgets:
   -
+    metric: TTFB
```

Comment:
Understand that it doesn't have much value on it's own. The reason I put it in was that, looking at the webpagetest results we've been getting to date, there is a huge correlation between this value and the ones we do care about (start render and speed index).  Having this value alert gives us a quick indication of where any problems may lie.